somma.py

```diff
@@ -0,0 +1,4 @@
+num1 = int(input("Inserisci il primo numero:  "))
+num2 = int(input("Inserisci il secondo numero:  "))
+somma = num1+num2
```

Comment:
Potresti definire una funzione `somma(num1, num2)` affinchè essa possa venire riutilizzata anche in futuro e non 'limitarla' ad un uso locale